### ISSUE TITLE
Remove the "_v1" suffix in models on OSF and on this repos

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/axondeepseg#278]
The versioning is now tracked by OSF, more info [here](https://github.com/neuropoly/axondeepseg/wiki/Template-for-Readme.txt)

**Comment by mariehbourget** [comment]
I’ve reviewed the Readme.txt for version tracking.
Everything is clear to me but I have a question about the steps 2 and 3 to train the model.
My understanding is that we want to be able the track the condition of training for each model.
In that case, is it important to document which images were used in the training set and which images were used in the validation set?

In the guide_dataset_building.ipynb of the master branch, the split is done manually before running the notebook.
In the add_split branch (#276) , there is a split 80-20% split with a seed in the notebook.

**Comment by mathieuboudreau** [comment]
Thanks @mariehbourget ! In the add_split branch, the way we can document/reproduce the split is in fact using the seed. For the same dataset, the seed should make it that the same exact 80:20 split of images will be reproduced - you can test this out. For a different split, the final training/validation sets will be different. I preferrred this way over doing it manually because when you do it manually, it can be difficult to record how you split it (and if you actually did it randomly..)

If you need more clarification, let me know and we could setup a video call.

**Comment by mariehbourget** [comment]
Thanks @mathieuboudreau !
Would it be important for the purpose of the Readme.txt to document the split percentages and the seed?

**Comment by vs74** [comment]
> I’ve reviewed the Readme.txt for version tracking.
> Everything is clear to me but I have a question about the steps 2 and 3 to train the model.
> My understanding is that we want to be able the track the condition of training for each model.
> In that case, is it important to document which images were used in the training set and which images were used in the validation set?
> 
> In the guide_dataset_building.ipynb of the master branch, the split is done manually before running the notebook.
> In the add_split branch (#276) , there is a split 80-20% split with a seed in the notebook.

@mariehbourget, I also believe that for steps 2 and 3 there should be some changes
```
2. Get the data: [Enter the OSF URL pointing to the data-- make sure to include the version]
3. Run the notebook "guide_dataset_building.ipynb" to perform a split into training and validation set. 
```

Because i think that the only thing which would change here is the models (fine tuning, new model implementation, etc..) not the data, so instead of pointing link to the data we should point  link to the newly trained models in the OSF storage. What do you think @jcohenadad  and @mathieuboudreau ?

**Comment by mathieuboudreau** [comment]
>Because i think that the only thing which would change here is the models (fine tuning, new model implementation, etc..) not the data, so instead of pointing link to the data we should point link to the newly trained models in the OSF storage

How can we add the updated link to the newly trained model when it hasn't been uploaded yet (it should be uploaded with the README file). Also, as long as it has the same name, the link will never change, only the version number on OSF.

**Comment by vs74** [comment]
Yeah, that makes sense. I will make the necessary changes. Thanks @mariehbourget for the feedback.

**Comment by vs74** [comment]
One question, though it's not related to this issue, the SEM raw dataset hosted on OSF is merely of size  **10 MB** whereas the raw dataset size of **TEM > 1GB**. Just curious, is the raw dataset of SEM correct ? 
NOTE: - I didn't validate the raw dataset of SEM on the OSF storage.

**Comment by mathieuboudreau** [comment]
Why'd I just get 10 alerts of data being uploaded on osf? If you're testing something, it shouldn't be done on the main models, you should use a fake model name that can then be deleted afterwards. OSF tracks every updated upload, so there's a lot of versions now that aren't clear why they exist, and will be difficult to understand in the future.

**Comment by mathieuboudreau** [comment]
> One question, though it's not related to this issue, the SEM raw dataset hosted on OSF is merely of size **10 MB** whereas the raw dataset size of **TEM > 1GB**. Just curious, is the raw dataset of SEM correct ?
> NOTE: - I didn't validate the raw dataset of SEM on the OSF storage.

Yes, it's correct. See paper for more details on the quantity and resolution of data for each model.

**Comment by vs74** [comment]
Sorry for that. I mistakenly deleted SEM main model on OSF ( thinking that in order to update SEM model,  I had to first delete SEM model and and then re upload it ). Doing this, the new SEM had a version of 1, for which I had to update the SEM model to synchronise with the TEM versioning which was 4. I  apologise for this mistake.

**Comment by mathieuboudreau** [comment]
@vs74 ok, thanks for clarifying. I looked into it, and there's no easy to "protect" files from being deleted. Also, OSF only asks for a quick confirmation ("Are you sure you want to delete this file?"), and doesn't really make it clear to the user that the files will be deleted permanently and that this is unreversible. Therefore, to protect the files currently on OSF, I've removed the write permissions for all users except @jcohenadad and I, so that this mistake doesn't happen again. If you need something uploaded to OSF, please contact me by email with a Google Drive link to the file and I'll arrange to do so ASAP.

**Comment by Stoyan-I-A** [comment]
Note for this Issue:
 [This test](https://github.com/neuropoly/axondeepseg/blob/bb96c56c8f5132449c98fb11c1179d86ccddc36f/test/test_ads_utils.py#L71) needs to be modified before closing this issue.

**Comment by vs74** [comment]
I modified the test in my branch.
    On Monday, 2 March 2020, 09:05:48 AM GMT+1, Stoyan I. Asenov <notifications@github.com> wrote:  
 
 
Note for this Issue:
This test needs to be modified before closing this issue.
